src/annotator/plugin/bucket-bar.js

```diff
@@ -22,44 +26,68 @@ function scrollToClosest(anchors, direction) {
 export default class BucketBar extends Delegator {
   constructor(element, options, annotator) {
     const defaultOptions = {
-      // gapSize parameter is used by the clustering algorithm
-      // If an annotation is farther then this gapSize from the next bucket
-      // then that annotation will not be merged into the bucket
-      // TODO: This is not currently used; reassess
-      gapSize: 60,
-      html: '<div class="annotator-bucket-bar"></div>',
       // Selectors for the scrollable elements on the page
-      scrollables: ['body'],
+      scrollables: [],
     };
 
     const opts = { ...defaultOptions, ...options };
-    super($(opts.html), opts);
 
+    const el = document.createElement('div');
+    el.className = 'annotator-bucket-bar';
+    super(el, opts);
+
+    this.annotator = annotator;
+
+    /** @type {Bucket[]} */
     this.buckets = [];
-    this.index = [];
-    this.tabs = $([]);
+    /** @type {HTMLElement[]} - Elements created in the bucket bar for each bucket */
+    this.tabs = [];
+
+    // The element to append this plugin's element to; defaults to the provided
+    // `element` unless a `container` option was provided
+    let container = /** @type {HTMLElement} */ (element);
 
     if (this.options.container) {
-      $(this.options.container).append(this.element);
-    } else {
-      $(element).append(this.element);
+      // If a container element selector has been provided, and there is an
+      // element corresponding to that container — use it
+      const containerEl = /** @type {HTMLElement | null } */ (document.querySelector(
+        this.options.container
+      ));
+      if (containerEl) {
+        container = containerEl;
+      } else {
+        // A container selector has been supplied, but it didn't pan out...
+        console.warn(
```

Comment:
This might be silly. I'm not sure how to handle an error condition here gracefully.